validation/scripts/test_discovery.py

```diff
@@ -7,6 +7,9 @@
 PTH = os.path.join(
     os.path.dirname(os.path.abspath(__file__)), "..", "test", "unpolarised"
 )
+POL_PTH = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), "..", "test", "polarised"
```

Comment:
It might make sense to change all `os.path --> pathlib` at some point in the future.